### PR TITLE
Console log protocol tests

### PR DESCRIPTION
**Pull request by asachin96**

### CODE REVIEW

**Comment by TCourtneyOwen** [comment]
Any ideas what's causing the log test to fail?

**Comment by asachin96** [comment]
> Any ideas what's causing the log test to fail?

There's a bug in Console redirection, which is I fixed in PR https://github.com/microsoft/react-native-windows/pull/2624. I'm removing type check till that gets merged to devmain

File: test/ui-test.ts [hunk 1]
```diff
@@ -31,6 +63,7 @@ describe("Test Excel Custom Functions", function () {
     describe("Get test results for custom functions and validate results", function () {
         it("should get results from the taskpane application", async function () {
             this.timeout(0);
+            initializeDebugger();
```
Comment (TCourtneyOwen) [inline test/ui-test.ts]: should we consider adding initializeDebugger to the UI tests for our other repos? Or is this only relevant to custom functions?

Comment (asachin96) [reply test/ui-test.ts]: Yeah, we can move to a common location. Where do u suggest?

Comment (TCourtneyOwen) [reply test/ui-test.ts]: I would suggest office-addin-debugging or office-addin-test-helpers. However I think make that a separate work item to move it there so your current work isn't blocked while waiting on moving this to a shared location

File: test/ui-test.ts [hunk 2]
```diff
@@ -5,6 +5,38 @@ import { AppType, startDebugging, stopDebugging } from "office-addin-debugging";
 import { pingTestServer } from "office-addin-test-helpers";
 import * as officeAddinTestServer from "office-addin-test-server";
 import * as path from "path";
+import * as WebSocket from "ws";
+
+let connected = false;
+let events = [];
+let retries = 10;
+
+function initializeDebugger(){
```
Comment (akrantz) [inline test/ui-test.ts]: Consider naming this `initializeMockDebugger()` or `startMockDebugger()`...

**Comment by akrantz** [comment]
Sachin, you can merge the latest changes from the master branch. We'll wait to merge this PR until we've completed the Yo Office release.

**Comment by asachin96** [comment]
> See comments.
> 
> @TCourtneyOwen Let's sync up tomorrow if we want to merge this yet or not, given the desire to update generator-office.
> 
> Also, if port 9229 is already in use, what happens when the test is run?

WebSocket will retry for 10 times to open if it doesn't succeed, nothing happens, but tests will fail because there will no events to validate

**Comment by lindalu-MSFT** [comment]
What is the status of this PR? Can it be closed?
